packages/sku/src/utils/polyfillRegistry.ts

```diff
@@ -0,0 +1,42 @@
+export interface PolyfillRegistryEntry {
+  reason: string;
+  docsUrl?: string;
+}
+
+export type PolyfillRegistry = Record<string, PolyfillRegistryEntry>;
+
+/**
+ * Registry of polyfills that are unnecessary for sku's supported browsers
+ */
+export const POLYFILL_REGISTRY: PolyfillRegistry = {
+  'core-js': {
+    reason:
+      'Most ES2015-ES2017 features are natively supported in modern browsers',
+    docsUrl: 'https://github.com/zloirock/core-js#usage',
+  },
+  '@babel/polyfill': {
+    reason: 'Deprecated package, replaced by core-js',
```

Comment:
This message might result in someone replacing `@babel/polyfill` with `core-js`, which is also something we don't recommend. Maybe we should just say it's deprecated and nothing else?